Error building on Windows with tsc 3.8.3 and node 16.1

$ npm install

> @microsoft.azure/autorest.testserver@3.0.26 prepublish
> npm run build


> @microsoft.azure/autorest.testserver@3.0.26 build
> tsc -p ./tsconfig.build.json

src/services/spec-coverage-service.ts:35:72 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(o: {}): string[]', gave the following error.
    Argument of type 'object | null' is not assignable to parameter of type '{}'.
      Type 'null' is not assignable to type '{}'.
  Overload 2 of 2, '(o: object): string[]', gave the following error.
    Argument of type 'object | null' is not assignable to parameter of type 'object'.
      Type 'null' is not assignable to type 'object'.

35         methods: content && typeof content === "object" ? (Object.keys(content) as HttpMethod[]) : [],
                                                                          ~~~~~~~



Found 1 error.

npm ERR! code 2
npm ERR! path C:\Users\daroch\Projects\AzureMaps\autorest.testserver
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c npm run build

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\daroch\AppData\Local\npm-cache\_logs\2021-07-08T11_07_30_115Z-debug.log

Content of the Debug Log:

[2021-07-08T11_07_30_115Z-debug.log](https://github.com/Azure/autorest.testserver/files/6784004/2021-07-08T11_07_30_115Z-debug.log)

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.testserver",
-  "version": "3.0.28",
+  "version": "3.0.29",
   "description": "Autorest test server.",
   "main": "dist/cli/cli.js",
   "bin": {
@@ -30,7 +30,7 @@
     "coverage-push": "node ./dist/reporter/app-immediate-push",
     "coverage-show": "node ./dist/reporter/app-show",
     "show-results": "node ./dist/reporter/app-show",
-    "prepublish": "npm run build"
+    "prepack": "npm run build"
   },
   "engines": {
     "node": ">=10"

